biome.jsonc

```diff
@@ -130,10 +89,10 @@
 	},
 	"formatter": {
 		"enabled": true,
-		"ignore": ["**/dist/**", "**/lib/**"],
+		"includes": ["**", "!**/dist/**", "!**/lib/**"],
 		"formatWithErrors": true,
 		"indentStyle": "tab",
-		"lineWidth": 95,
+		"lineWidth": 120,
```

Comment:
Add a comment that this change is to minimize churn; we can discuss more in further code review.